### DISCUSSION TITLE
Do we like Discussions?

### DISCUSSION THREAD

**Discussion by PeerRich** [NostrGit/NostrGit#95] [General]
At cal.com (https://github.com/calcom/cal.com) we decided to have everything in issues so you have a mono-view of everything.

I am already feeling like some people dont find Discussions when visiting.

How do we feel about Issues vs Discussions?

**Comment by PeerRich** [comment]
Happy to disable discussions again and use the "discussion" label

**Reply by cypherhoodlum** [reply]
I feel your pain as I watch you convert each issue into a discussion :smile: I'm okay with using issues only :+1:

**Reply by cypherhoodlum** [reply]
Actually not only okay with it, but I strongly prefer seeing a mono-view.

**Reply by cypherhoodlum** [reply]
@brycedev @pierresisson @melvincarvalho @BruceWangyq  what do you think? I could start converting the discussions back to issues if most approve.

**Reply by cypherhoodlum** [reply]
If there are no replies by tomorrow, I will interpret it as @PeerRich mentioned; these discussions don't reach people the way issues do.

**Reply by BruceWangyq** [reply]
> @brycedev @pierresisson @melvincarvalho @BruceWangyq what do you think? I could start converting the discussions back to issues if most approve.

Let's go with @PeerRich suggestion. Personally, I don't use discussions much.

**Reply by pierresisson** [reply]
@PeerRich i totally agree, i read issue/pr but not discussion so much

**Reply by cypherhoodlum** [reply]
Okay! Staring to convert them back into issues now.

**Reply by cypherhoodlum** [reply]
Turns out it's not so straightforward as turning issues into discussions. Added #113 for everyone to see.
